vm/thread.go

```diff
@@ -162,7 +162,7 @@ func (t *thread) retrieveBlock(fileName, blockFlag string) (blockFrame *normalCa
 	return
 }
 
-func (t *thread) sendMethod(methodName string, argCount int, blockFrame *normalCallFrame, instruction *instruction) {
+func (t *thread) sendMethod(methodName string, argCount int, blockFrame *normalCallFrame, sourceLine int) {
```

Comment:
@saveriomiroddi You can use this function to call Goby method inside Go. You should pass the method's name and the number of arguments you provided. Leave `blockFrame` to `nil`, and `sourceLine` to be `-1`.

And the most important thing is you need to push every arguments your want to use on the stack in order. This can't be simplify because builtin method and normal methods have different ways to retrieve arguments object from stack.